.travis.yml

```diff
@@ -61,13 +61,17 @@ jobs:
       script:
         - unbuffer node build-system/pr-check/validator-tests.js
     - stage: build
-      name: 'Dist, Bundle Size'
+      name: 'Nomodule Build'
       script:
-        - unbuffer node build-system/pr-check/dist-bundle-size.js
+        - unbuffer node build-system/pr-check/nomodule-build.js
     - stage: build
-      name: 'Module Dist, Bundle Size'
+      name: 'Module Build'
       script:
-        - unbuffer node build-system/pr-check/module-dist-bundle-size.js
+        - unbuffer node build-system/pr-check/module-build.js
+    - stage: test
+      name: 'Bundle Size'
+      script:
+        - unbuffer node build-system/pr-check/bundle-size.js
```

Comment:
> does it matter what order it appears in the list?

The way you've done it LGTM. (This job will be the first to start in the test stage, so we'll see results ASAP.)